gems/quilt_rails/dev.yml

```diff
@@ -4,7 +4,7 @@ up:
   - bundler
   - node:
       yarn: v1.19.1
-      version: v10.18.1
+      version: v12.14.0
```

Comment:
`version` coming after `yarn` is giving me a cognitive dissonance :/ Could we reorder them while we're at it?